drafts/proof-of-spacetime.md

```diff
@@ -1,24 +1,141 @@
 ## Proof-of-Spacetime
-   
-This document descibes 
- 
+
+This document descibes
+
+- Base-PoSt
 - VDF-PoSt: a Proof-of-Spacetime using VDFs
 - An Extension to PoSt to support multiple sectors
-- An Extension to PoSt to support challenges taken from a Random Beacon 
+- An Extension to PoSt to support challenges taken from a Random Beacon
 
 ## Syntax
 
 - **PoSt Epoch**: The total time passing between Online PoReps in the PoSt Computation (in VDF, this interval is the time it takes to run a VDF and an Online PoRep prove step). We define the number of epochs as `POST_EPOCHS_COUNT`
-- **PoSt Period**: The total time it takes to run a single PoSt. If a PoSt is repeated multiple times, we define the number of periods as `POST_PERIODS_COUNT`. This can be reasoned as: `PoSt Epoch * POST_EPOCHS_COUNT`. We assume that the best  Post Period time is `MIN_POST_PERIOD_TIME` 
+- **PoSt Period**: The total time it takes to run a single PoSt. If a PoSt is repeated multiple times, we define the number of periods as `POST_PERIODS_COUNT`. This can be reasoned as: `PoSt Epoch * POST_EPOCHS_COUNT`. We assume that the best  Post Period time is `MIN_POST_PERIOD_TIME`
 - **Total proving time**: The time it takes to run a PoSt. Note that a PoSt could be a composition of multiple PoSt. This can be reasoned as: `PoSt Period * POST_PERIODS_COUNT` TODO Check with papers' syntax
 
+
+## Base-PoSt
+
+- **Setup Parameters**
+  - Same as Public parameters.
+- **Public Parameters**
+  - `post_periods_count`: uint
+  - `challenge_count`: number of challenges to be asked at each iteration
+  - `sector_size`: size of the sealed sector in bytes
+  - `sectors_count`: number of sectors over which the proof is performed
+  - `challenge_bits`: number of bits in one challenge (length of a merkle path)
+  - `seed_bits`: number of bits in one challenge
+- **Public Inputs**
+  - `CommRs : [sectors_count]Hash` CommRs must be ordered by their timestamp on the blockchain.
+- **Private Inputs**
+  - `replicas: [sectors_count]SealedSector`: sealed sectors
+- **Proof**
+  - `post_proofs [post_periods_count]PoRepProof`
+
+### Methods
+
+#### `ProveStep(Step, Beacon_t, PublicParameters, PublicInputs, PrivateInputs)`
```

Comment:
What is a step? How does the caller of `ProveStep` obtain a value to be used as the `Step` parameter here?

Reply:
no steps anymore